app/controllers/twitter-searcher.controller.js

```diff
@@ -0,0 +1,74 @@
+(function() {
+  'use strict';
+
+  class TwitterSearcherCtrl {
+    constructor($scope, DataCreatorService, SearchingService) {
+      Object.assign(this, {$scope, DataCreatorService, SearchingService});
+      this.tweets = [];
+      this.words = [{

```

Comment:
dlaczego uzywasz czegos takiego? { id: 1 }?

Reply:
każdy input ma przypisany swój nr id, w razie gdyby ktoś zmienił drugi input to podmieniają się tweety w drugim okienku z listy, druga kolumna itd